integrations/localhero/pom.xml

```diff
@@ -47,7 +49,10 @@
             <artifactId>feign-slf4j</artifactId>
             <version>${feign.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.h2database</groupId>
```

Comment:
Why do we need a database? For the batch? Afaik, it creates a map based storage when no datasource is defined. I went without it on purpose.

Reply:
I never used the complete in memory approach but used a in-memory h2 instead -> but you are right I'll revert these db related changes